tests/foreman/ui/test_contenthost.py

```diff
@@ -1313,3 +1315,39 @@ def test_pagination_multiple_hosts_multiple_pages(session, module_host_template)
         # Assert that total items reported is the number of hosts created for this test
         total_items_found = pagination_values['total_items']
         assert int(total_items_found) >= host_num
+
+
+@tier3
+def test_search_for_virt_who_hypervisors(session):
+    """
+    Search the virt_who hypervisors with hypervisor=True or hypervisor=False.
+
+    :id: 3c759e13-d5ef-4273-8e64-2cc8ed9099af
+
+    :expectedresults: Search with hypervisor=True and hypervisor=False gives the correct result.
+
+    :BZ: 1653386
+
+    :CaseLevel: System
+
+    :CaseImportance: Medium
+    """
+    org = entities.Organization().create()
+    with session:
+        session.organization.select(org.name)
+        assert not session.contenthost.search("hypervisor = true")
+        # create virt-who hypervisor through the fake json conf
+        data = hypervisor_json_create(hypervisors=1, guests=1)
+        hypervisor_name = list(data.keys())[0]
+        owner = f"owner={org.name}&env=Library"
+        url = f'https://{settings.server.hostname}/rhsm/hypervisors?{owner}'
```

Comment:
The API you are using is deprecated,  you may try to use this async API to instead
https://${sat_host}/rhsm/hypervisors/Default_Organization

Reply:
I have tried this and created a PR about this: https://github.com/SatelliteQE/robottelo/pull/7916